package.json

```diff
@@ -35,6 +35,10 @@
     "contributors:generate": "all-contributors generate",
     "heroku-postbuild": "yarn build && yarn workspace @keystonejs/demo-project-blog build"
   },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/keystonejs/keystone.git"
+  },
```

Comment:
We will need to have this snipped in all the `package.json` files for all of the packages within the monorepo.